CHANGELOG.md

```diff
@@ -66,6 +66,10 @@
 - Added support for `Series.dt.is_leap_year`.
 - Added support for `DataFrame.items`.
 - Added support for `Series.dt.floor` and `Series.dt.ceil`.
+- Added support for `DatetimeIndex` properties: `year`, `month`, `day`, `hour`, `minute`, `second`, `microsecond`,
```

Comment:
Can we add the API docs here docs/source/modin/indexing.rst similar to what pandas does: https://pandas.pydata.org/docs/reference/api/pandas.DatetimeIndex.html

Reply:
I see there is separate JIRA to take care of this SNOW-1570717.